gh-pages/use-baseline.md

```diff
@@ -0,0 +1,21 @@
+---
+permalink: use-baseline
+description: How to use the Baseline in your projects.
+---
+
+# Use Baseline in your project
+
+You can use Baseline to:
+
+- Make decisions about what web platform features to use in your sites and applications
+- Tell others whether features work across browsers
+
+## Check if a feature works across browsers
+
+When visiting sites like [Can I Use](https://caniuse.com), [MDN Web Docs](https://developer.mozilla.org/), [RUM Archive Insights](https://rumarchive.com/insights/#baseline) and [webstatus.dev](https://webstatus.dev), check the status of a feature to learn if it works across browsers.
```

Comment:
Do you have a suggestion for how to adjust this?

Reply:
```suggestion
When visiting sites like [Can I Use](https://caniuse.com), [MDN Web Docs](https://developer.mozilla.org/), and [RUM Archive Insights](https://rumarchive.com/insights/#baseline) check the status of a feature to learn if it's implemented across browsers.
```

Reply:
> This tells us if it's _implemented_ across browsers, not whether it _works_. Widely available features are assumed to have good interop, but it's possible (though unfortunate) that newly available features still have notable differences.

A few points on this, @jgraham:

- The bar is higher than implemented because we ignore implementations behind flags, in future-dated releases, or using (unspecified) prefixes and aliases.
- We don't assume good interop on a timeline; the compute-baseline code actually checks what BCD reports for every release in the 30-month range and rejects releases annotated with, for example, the `partial_implementation` flag. And we do some human review. For example, I check the notes when updating features, so that (for example) BCD hasn't mistakenly allowed a partial through in a note without the requisite flag.

I'm not committed to "works" per se, but I think we can make a stronger and more accurate claim than to "implementation." Whatever the terminology, i we report a feature as Baseline and it's not working, I'd really like to hear about it. That merits an editorial override here and (very probably) a fix upstream to BCD.

Reply:
"available" would also be fine.

Reply:
Fixed in de35a122eee9e74ee11fd673eb1d4bb7c8321a7c